src/prototype_room_my.js

```diff
@@ -292,7 +292,7 @@ Room.prototype.executeRoom = function() {
   const nextroomers = this.findPropertyFilter(FIND_MY_CREEPS, 'memory.role', ['nextroomer'], false, {
     filter: object => object.memory.base !== this.name
   });
-  const building = nextroomers.length > 0 && this.controller.level < 5;
+  const building = nextroomers.length > 0 && this.controller.level < 4;
```

Comment:
Within the last months, I improved a lot on RCL < 5. I think low rooms are now properly capable of being more effective than the `nextroomers`. I'm even thinking to reduce this more, send nextroomers only to RCL 3, but send energy to the new room.